src/_data/sites/es.yml

```
@@ -0,0 +1,570 @@
+#------------------------------------------------------------------------------
+# Spanish Site Details
+# The primary ESLint site that is hosted at eslint.org
+# Author: Luis Morón
+#------------------------------------------------------------------------------
+
+
+#------------------------------------------------------------------------------
+# Global Settings
+#------------------------------------------------------------------------------
+
+language:
+  code: es
+  flag: 🇪🇸
+  name: Spanish (ES)
+locale: es-ES
+hostname: new.eslint.org
```

We need to set up a unique host name here. 


```suggestion
hostname: es.eslint.org
```